netdata.spec.in

```diff
@@ -42,15 +47,19 @@ Requires(pre): /usr/sbin/groupadd
 Requires(pre): /usr/sbin/useradd
 
 %if %{with systemd}
+%if 0%{?suse_version}
+Requires(preun):  systemd-rpm-macros
+Requires(postun): systemd-rpm-macros
+Requires(post):   systemd-rpm-macros
+%else
```

Comment:
instead of this conditional, please define a macro at prologue with the package name to depend, the less conditionals we have in spec body the easier to maintain.